docs/src/explanations/connectable.md

```diff
@@ -543,6 +543,33 @@ This generates the following Verilog, where `p` is implicitly truncated prior to
 getVerilogString(new Example14)
 ```
 
+### Excluding members from any operator on a Connectable
+
+If a user wants to always exclude a field from a connect, use the `exclude` mechanism which will never connect the field (as if it didn't exist to the connection).
```

Comment:
Is my RTFC correct, that this one will still error if they mismatch? Like, you can only exclude things if they *would* have matched (unless you waived or squeezed or otherwise). This might be worth clarifying with a word or two, i see you made it more clear in the other sections.